_projects/gitmate_jira.md

```diff
@@ -43,7 +42,6 @@ requirements:
   - "The participant should have at least one patch to
      [GitMate](https://gitlab.com/gitmate/open-source/gitmate-2) accepted."
   - "The participant should demonstrate that he can operate Jira from the Python shell in his proposal."
-status: in_progress
```

Comment:
@nkprince007 , what is the status of this project?

Reply:
This project never surfaced completion. During the development of this project, we got the award from Mozilla Open Source Support and so, we shifted our focus towards finishing MOSS stuff.

Reply:
Well, more like stopped amidst. No one seems to care about adding support to Jira because it involves having a lot of stuff to develop. First off, it requires our own instance of Jira to begin with.

Reply:
Ok, I'll change the project status to "available". Is that okay?

Reply:
Yes.